src/playlist-loader.js

```diff
@@ -619,6 +613,8 @@ export default class PlaylistLoader extends EventTarget {
       return;
     }
 
+    this.updateMediaUpdateTimeout_(refreshDelay(playlist, true));
```

Comment:
May be worth a comment above here saying why we update the timeout even if it's switching to the active playlist

Reply:
added